script/zip_manifests/dist_zip.win.arm64.manifest

```diff
@@ -0,0 +1,70 @@
+LICENSE
+LICENSES.chromium.html
+chrome_100_percent.pak
+chrome_200_percent.pak
+electron.exe
```

Comment:
@richard-townsend-arm can you clarify whether we need to package `d3dcompiler_47.dll` ? Based on the context from https://bugs.chromium.org/p/chromium/issues/detail?id=893460#c64 , it seems that for ARM64 it relies on the library from system sdk ?

Reply:
In the CI environment I have deployed within Arm, we are not packaging d3dcompiler_47.dll.